### DISCUSSION TITLE
Linking two quasi-static 3 phase AC circuits using controlled voltage source & current load

### DISCUSSION THREAD

**Discussion by svarig** [modelica/ModelicaStandardLibrary#3880] [Q&A]
Can the MSL Electrical quasi-static single phase and polyphase circuit and component models can be used to link two separate circuits via  complex signal interfaces (Modelica.ComplexBlocks.Interfaces.ComplexInput) or real signals and signal controlled source/load models. 

Example use cases include graphical/schematic based implementation of 

1) transformer primary & secondary circuits that are not directly connected in electrical domain as one circuit, 
2) averaged power converter models between a quasi-static and a DC circuit or two quasi-static circuits

The example model cited, Modelica.Electrical.QuasiStatic.SinglePhase.Utilities.IdealACDCConverter, implements teh AC-DC conversion in code rather than using circuit schematics with controlled voltage source (on DC side) and controlled current load (on QS AC side). 

If possible, it would be great to include some examples that show how to do this. If there is a limitation in doing that, perhaps the documentation can be updated to describe these limitations.  

During my testing, I am getting errors related to the reference angle in the quasi-static connectors. It appears that a signal driven sources cannot be added to a circuit that already has another source. I can provide the test model if needed. 
 
Appreciate clarifications...!

**Comment by dietmarw** [comment]
I'm just including @AHaumer and @christiankral  here since I'm not sure if they get notifications on the discussions here.
